_posts/en/newsletters/2024-01-10-newsletter.md

```diff
@@ -0,0 +1,288 @@
+---
+title: 'Bitcoin Optech Newsletter #284'
+permalink: /en/newsletters/2024/01/10/
+name: 2024-01-10-newsletter
+slug: 2024-01-10-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Discussion about proposed v3 transaction relay policy:**
```

Comment:
I might be biased, but perhaps a more suitable title might be "Discussion about LN anchors and v3 proposal." I think the post was originally created to move out-of-scope discussion (especially the criticisms of CPFP and LN anchors in general) away from the PR.